dashboard/test/models/script_test.rb

```diff
@@ -327,7 +327,7 @@ def populate_cache_and_disconnect_db
   end
 
   test 'level_concept_difficulty uses preloading' do
-    level = Script.find_by_name('20-hour').script_levels.third.level
```

Comment:
Ah, so the test wasn't testing what it should be. Please change to a level that does have a level_concept_difficulty (see the name of the test).